### DISCUSSION TITLE
是否需要取消 Github Action "main" 觸發分支？

### DISCUSSION THREAD

**Discussion by FizzyElt** [FizzyElt/functional-programming#76] [Show and tell]
有些不是文章的修改併到 main 還是會跑 Action，感覺沒什麼必要
依然都是併到 main，如果是文章會再併到 forester 做 deploy

**Comment by dannypsnl** [comment]
加 file pattern 就可以搞定了

**Comment by FizzyElt** [comment]
https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#using-filters-to-target-specific-paths-for-pull-request-or-push-events
應該是這個

**Reply by dannypsnl** [reply]
沒錯，我通常都設定白名單啦
